### PR TITLE
Prepare and show changelog

### PR DESCRIPTION
**Pull request by suricactus**
Implements the feature request in #1192 . This PR uses the Markdown instead of HTML (the gif in the issue). The changelog is loaded once on startup and cached until the app is restarted. Please write stricter markdown in the release notes, all the newlines matter.

Peek 2020-08-20 18-04

### CODE REVIEW

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1249-5e02e7e3ec9d2f4be863e461239d1bfaf995ce6f-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1249-5e02e7e3ec9d2f4be863e461239d1bfaf995ce6f-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1249-5e02e7e3ec9d2f4be863e461239d1bfaf995ce6f-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1249-5e02e7e3ec9d2f4be863e461239d1bfaf995ce6f-x86_64.apk)

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1249-56e4c826e027176c5f8ece4f280bdd916d5f66c5-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1249-56e4c826e027176c5f8ece4f280bdd916d5f66c5-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1249-56e4c826e027176c5f8ece4f280bdd916d5f66c5-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1249-56e4c826e027176c5f8ece4f280bdd916d5f66c5-x86_64.apk)

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1249-785da6919c1812d0d8ac7e3781d5927322f85195-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1249-785da6919c1812d0d8ac7e3781d5927322f85195-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1249-785da6919c1812d0d8ac7e3781d5927322f85195-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1249-785da6919c1812d0d8ac7e3781d5927322f85195-x86_64.apk)

**Comment by signedav** [comment]
The link still does not work. But it seems (but not 100% sure) that the change log does not have the focus. This feeling I have, because on a back key press the About dialog closes completely, so I assume this is triggered: https://github.com/opengisch/QField/blob/master/src/qml/qgismobileapp.qml#L1491-L1496

Maybe you need to add `Component.onCompleted: focusstack.addFocusTaker( this )` to the changelog instance.
And maybe the part mentioned above as well.

**Comment by suricactus** [comment]
Testing on desktop and mobiles shows the links are working, even though there might be some delay on mobile.

The issue with back button was existing though, now it should close only the popup.

**Comment by signedav** [comment]
You are right regarding the links. It's working.

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1249-f6dce5f85b27cc2c4af9ef435501c6067b203b70-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1249-f6dce5f85b27cc2c4af9ef435501c6067b203b70-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1249-f6dce5f85b27cc2c4af9ef435501c6067b203b70-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1249-f6dce5f85b27cc2c4af9ef435501c6067b203b70-x86_64.apk)

File: .tx/config
Comment (m-kuhn) [inline .tx/config:3]: @suricactus can you create a separate PR for this one so we get back to green and can wait for a review by @signedav and/or @nirvn of this one?

Comment (suricactus) [reply .tx/config:3]: You were way too fast :)

**Comment by signedav** [comment]
I cannot open the app. Neither I can open the msster build. I'll take a look this afternoon...

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1249-c0499a7efe040401eca3bc2f82cb1895732f04eb-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1249-c0499a7efe040401eca3bc2f82cb1895732f04eb-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1249-c0499a7efe040401eca3bc2f82cb1895732f04eb-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1249-c0499a7efe040401eca3bc2f82cb1895732f04eb-x86_64.apk)

**Review by signedav — APPROVED** [review]
Looks good. Thanks!

**Comment by VxTedxV** [comment]
Do I have to open an issue to tell you that it shows big spaces **before** the last line of each paragraph in QField 1.7.0 ?

**Comment by suricactus** [comment]
Can you post a screenshot there?

**Comment by VxTedxV** [comment]
ezgif com-video-to-gif

**Comment by signedav** [comment]
Maybe it's something in the release markdown. I used HTML there for example...

**Comment by signedav** [comment]
@suricactus you just fixed it by inserting `[]()`?

**Comment by suricactus** [comment]
Should be fixed, at least for me it looks fine. @VxTedxV can you try to open the changelog once again (About -> Changelog) and confirm it no longer looks weird.

**Comment by VxTedxV** [comment]
I did and looked the same as on my video.

**Pull request merged**
